.github/workflows/main.yml

```diff
@@ -167,32 +136,116 @@ jobs:
         run: |
           yum install -y java-1.8.0-openjdk-devel
 
-      - name: build zetasql
+      - name: build zetasql parser
         run: |
           source /opt/rh/devtoolset-8/enable
           source /opt/rh/rh-python38/enable
-          bazel build ${{ env.build_argv }} ${{ env.target }}
-
-      - name: Find Orphan Library
+          bazel query 'deps(${{ env.target }})' |grep //zetasql|xargs bazel build ${{ env.build_argv }} 
+          bazel build "@com_googleapis_googleapis//:all" ${{ env.build_argv }}
+          bazel build "@com_google_file_based_test_driver//file_based_test_driver:all" ${{ env.build_argv }}
+          bazel build "@com_googlesource_code_re2//:re2" ${{ env.build_argv }}
+      
+      - name: Test
         run: |
-          ./find_orphan_so.sh
+          bazel test ${{ env.build_argv }} --test_summary=detailed ${{ env.target }}
```

Comment:
Here we find a way to `bazel build //zetasql/parser:all` and to build its dependent static libraries at the same time